src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityOperatorBase.cs

```diff
@@ -58,10 +58,10 @@ protected sealed override (TypeWithAnnotations ReturnType, ImmutableArray<Parame
                     Parameters: ImmutableArray.Create<ParameterSymbol>(
                                     new SourceSimpleParameterSymbol(owner: this,
                                                                     TypeWithAnnotations.Create(ContainingType, NullableAnnotation.Annotated),
-                                                                    ordinal: 0, RefKind.None, "r1", isDiscard: false, Locations),
+                                                                    ordinal: 0, RefKind.None, "left", isDiscard: false, Locations),
```

Comment:
>"left" [](start = 94, length = 6)

Please add a test to `RecordTests` that demonstrates the change. Perhaps:
```csharp
var source = "record R;";
var comp = CreateCompilation(source);
Assert.Equal("System.Boolean R.op_Equality(R? left, R? right)", comp.GetMember("R.op_Equality").ToTestDisplayString());
Assert.Equal("System.Boolean R.op_Inequality(R? left, R? right)", comp.GetMember("R.op_Inequality").ToTestDisplayString());
```

Reply:
Never mind, it looks like there are a number of existing tests that should be affected.

---
In reply to: [597302570](https://github.com/dotnet/roslyn/pull/51973#discussion_r597302570) [](ancestors = 597302570)

Reply:
I think I have fixed all of the affected tests.